### PR TITLE
Update database back and restore procedures using smdba

### PR DESCRIPTION
**Pull request by rjmateus**
# Description

Update the back and restore for database using the smdba tool.

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 5.0

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Comment by rjmateus** [comment]
ping @mbussolotto

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+- Update database back and restore procedures using smdba
```
Comment (keichwa) [inline CHANGELOG.md:1]:
```suggestion
- Updated database backup and restore procedures using smdba  in Administration
  Guide
```

**Review by keichwa — CHANGES_REQUESTED** [review]

**Review by mbussolotto — APPROVED** [review]

File: modules/administration/pages/backup-restore.adoc [hunk 2]
```diff
@@ -225,21 +235,64 @@ We recommend you schedule regular database backups for a low-traffic period, to
 [IMPORTANT]
 ====
 Ensure you have at least three times the current database size of space available for backups.
-You can check your current database size by navigating to [path]``/var/lib/pgsql/`` and running [command]``df -h``.
+You can check your current database size by running [command]``mgrctl exec "df -h /var/lib/pgsql"``.
```
Comment (keichwa) [inline modules/administration/pages/backup-restore.adoc:238]:
```suggestion
You can check your current database size by running:

----
mgrctl exec "df -h /var/lib/pgsql"
----
```

File: modules/administration/pages/backup-restore.adoc [hunk 3]
```diff
@@ -225,21 +235,64 @@ We recommend you schedule regular database backups for a low-traffic period, to
...
 ====
 
+
+.Procedure: Create a directory for the backup (do it only once)
+. Enter in the container terminal
```
Comment (keichwa) [inline modules/administration/pages/backup-restore.adoc:243]:
```suggestion
.Procedure: Creating a directory for the backup (do it only once)

. In the container terminal, enter:
```

File: modules/administration/pages/backup-restore.adoc [hunk 4]
```diff
@@ -225,21 +235,64 @@ We recommend you schedule regular database backups for a low-traffic period, to
...
++
+----
+mgrctl term
+----
+. In your backup location, create a directory for the backup:
++
+As root:
```
Comment (keichwa) [inline modules/administration/pages/backup-restore.adoc:250]:
```suggestion
. In your backup location, create a directory for the backup.
  As root, enter:
```

File: modules/administration/pages/backup-restore.adoc [hunk 5]
```diff
@@ -225,21 +235,64 @@ We recommend you schedule regular database backups for a low-traffic period, to
...
+. Create a system unit file at [path]``/etc/systemd/system/server-db-backup.service`` with the content:
++
+----
+[Unit]
+Description="Server backup service"
+
+[Service]
+ExecStart=mgrctl exec "smdba backup-hot --enable=on --backup-dir=/var/spacewalk/db-backup"
+----
+. Create a system timer file at [path]``/etc/systemd/system/server-db-backup.timer`` with the content:
```
Comment (keichwa) [inline modules/administration/pages/backup-restore.adoc:272]:
```suggestion
. Create a systemd timer file at [path]``/etc/systemd/system/server-db-backup.timer`` with the content:
```

File: modules/administration/pages/backup-restore.adoc [hunk 6]
```diff
@@ -251,6 +304,11 @@ The [command]``smdba`` tool can be used to restore from backup in the case of fa
 
 
 .Procedure: Restoring from Backup
+. Enter in the container terminal
```
Comment (keichwa) [inline modules/administration/pages/backup-restore.adoc:307]:
```suggestion
. In the container terminal, enter:
```

**Review by keichwa — APPROVED** [review]

**Pull request merged**